### PR TITLE
Fixed update of print speed value in PrintingMenu

### LINKED ISSUE #1048: [BUG] Update the print speed value when printing
If I change the print speed while printing (for example, with an encoder directly in Marlin), the speed will not change on the PrintingMenu screen until I open the Speed Control menu. Only then will the Speed value be updated.
If it does not print, the StatusScreen screen works fine

https://youtu.be/kRYwBqbGrRU

### PR DESCRIPTION
**Pull request by radek8**
Fixed update of print speed value in PrintingMenu

Resolves #1048 

PR #1051 only worked when heating the pad and hotend. The print speed value was no longer updated after the print head started moving.
PR #1065 works all the time

### CODE REVIEW

File: TFT/src/User/Menu/PrintingMenu.c [hunk 1]
```diff
@@ -261,6 +261,7 @@ void toggleinfo(void)
       reDrawFan(FAN_ICON_POS);
     }
 
+    storeCmd("M220\nM221\n");
```
Comment (guruathwal) [inline TFT/src/User/Menu/PrintingMenu.c:264]:
```suggestion
    speedQuery();
```
Use speedQuery(); to ensure the next query is not added to buffer before the first command is sent, and also reduce code size.

Comment (radek8) [reply TFT/src/User/Menu/PrintingMenu.c:264]: I've already tried that in request # 1051
It only works if the hotend and bed are being heated.
As soon as G-codes start to be sent to move the head, the speed icon stops updating.
There is probably a problem with the condition of the speedQuery ();

Comment (radek8) [reply TFT/src/User/Menu/PrintingMenu.c:264]: @guruathwal  you have a suggestion on how to modify speedQuery (); to work in this case as well?

Comment (radek8) [reply TFT/src/User/Menu/PrintingMenu.c:264]: @guruathwal your change proposal only works if I also adjust speedQuery this way. Do you agree?

image

Comment (guruathwal) [reply TFT/src/User/Menu/PrintingMenu.c]: I will check and revert to you.

Comment (guruathwal) [reply TFT/src/User/Menu/PrintingMenu.c]: @radek8 the implementation of infohost.wait needs work

Comment (radek8) [reply TFT/src/User/Menu/PrintingMenu.c]: I don't understand your answers.
under the condition (infoHost.wait==false), updating the print speed value does not work correctly.
Do you know another way to solve it?
